app/shared/containers/Tools.js

```diff
@@ -42,17 +42,17 @@ class ToolsContainer extends Component<Props> {
         menuItem: t('tools_menu_keys'),
         render: () => <Tab.Pane><ToolsKeys {...this.props} /></Tab.Pane>,
       },
+      {
+        menuItem: t('tools_menu_proxy'),
+        render: () => <Tab.Pane><ToolsProxy {...this.props} /></Tab.Pane>,
+      },
       {
         menuItem: <Menu.Header className="ui">{t('tools_menu_state_header')}</Menu.Header>
       },
       {
         menuItem: t('tools_menu_state'),
         render: () => <Tab.Pane><ToolsStateWallet {...this.props} /></Tab.Pane>,
       },
-      {
```

Comment:
This was put here by mistake, right?

Reply:
Yeah - bad merge. I also am moving it into a new section of it's own.